lib/config_builder/model/trigger.rb

```diff
@@ -0,0 +1,29 @@
+# Vagrant SSH credential model
+#
+# @see http://docs.vagrantup.com/v2/vagrantfile/ssh_settings.html
```

Comment:
Incorrect as in the url is broken (worked for me) or incorrect as its pointing to the wrong content?